config/sync/user.role.site_builder.yml

```diff
@@ -0,0 +1,117 @@
+uuid: 959f5492-24df-4212-9a9c-a0da88d464c4
+langcode: en
+status: true
+dependencies: {  }
+id: site_builder
+label: 'Site Builder'
+weight: -1
+is_admin: null
+permissions:
+  - 'access administration pages'
+  - 'access config_pages overview'
+  - 'access content overview'
+  - 'access contextual links'
+  - 'access devel information'
```

Comment:
Probably don't want any devel in any of these permissions. the module isn't on anything other than local.